tests/stress/stress_test.go

```diff
@@ -61,127 +49,137 @@ func TestMain(m *testing.M) {
 	utils.Logger.Patch(log.SetLevel(logLvl))
 	logger.Patch(log.SetLevel(logLvl))
 
-	utils.GenerateGenesisThreeAuth()
-
 	// Start all tests
 	code := m.Run()
 	os.Exit(code)
 }
 
 func TestRestartNode(t *testing.T) {
-	numNodes := 1
-	nodes, err := utils.InitNodes(numNodes, utils.ConfigDefault)
-	require.NoError(t, err)
+	const numNodes = 1
+	defaultConfig := config.Default()
+	nodes := node.MakeNodes(t, numNodes, defaultConfig)
 
-	err = utils.StartNodes(t, nodes)
+	err := nodes.Init(context.Background())
 	require.NoError(t, err)
 
-	errList := utils.StopNodes(t, nodes)
-	require.Len(t, errList, 0)
+	ctx, cancel := context.WithCancel(context.Background())
 
-	err = utils.StartNodes(t, nodes)
-	require.NoError(t, err)
+	runtimeErrors, startErr := nodes.Start(ctx)
+	if startErr != nil {
+		cancel()
+		for _, runtimeError := range runtimeErrors {
+			<-runtimeError
+		}
```

Comment:
could you put this in a helper if you don't actually care?

Reply:
Done!